app/modules/intelligence/tools/tool_router.py

```
@@ -1,6 +1,6 @@
 from typing import List
 
-from fastapi import APIRouter, Depends, Header, HTTPException, Query
+from fastapi import APIRouter, Depends, Header, HTTPException
```

Ruff (0.8.2)

`fastapi.Header` imported but unused

Remove unused import

(F401)